test/runtests.jl

```diff
@@ -3,135 +3,135 @@ using Documenter
+    @test isapprox(sqrt(4pi), sqrt4π)
+    @test isapprox(sqrt(pi/2), sqrthalfπ)
+    @test isapprox(sqrt(1/2), invsqrt2)
+    @test isapprox(sqrt(1/(pi)), invsqrtπ)
+    @test isapprox(sqrt(1/(2pi)), invsqrt2π)
 end
 
 @testset "log" begin
-  @test isapprox(log(1/2), loghalf)
-  @test isapprox(log(2), logtwo)
-  @test isapprox(log(10), logten)
-  @test isapprox(log(pi), logπ)
-  @test isapprox(log(2pi), log2π)
-  @test isapprox(log(4pi), log4π)
+    @test isapprox(log(1/2), loghalf)
+    @test isapprox(log(2), logtwo)
+    @test isapprox(log(10), logten)
+    @test isapprox(log(pi), logπ)
+    @test isapprox(log(2pi), log2π)
+    @test isapprox(log(4pi), log4π)
 end
 
 @testset "type system" begin
-  @test twoπ === IrrationalConstants.Twoπ()
-  @test twoπ isa IrrationalConstants.IrrationalConstant
-  @test twoπ isa AbstractIrrational
+    @test twoπ === IrrationalConstants.Twoπ()
+    @test twoπ isa IrrationalConstants.IrrationalConstant
+    @test twoπ isa AbstractIrrational
 end
 
 @testset "hash" begin
-  for i in (twoπ, invπ, sqrt2, logtwo)
-    for j in (twoπ, invπ, sqrt2, logtwo)
-      @test isequal(i==j, hash(i)==hash(j))
+    for i in (twoπ, invπ, sqrt2, logtwo)
+        for j in (twoπ, invπ, sqrt2, logtwo)
+            @test isequal(i==j, hash(i)==hash(j))
+        end
     end
-  end
 end
 
 @testset "doctests" begin
-  DocMeta.setdocmeta!(
+    DocMeta.setdocmeta!(
     IrrationalConstants, :DocTestSetup, :(using IrrationalConstants); recursive=true
-  )
-  doctest(IrrationalConstants; manual=false)
+    )
+    doctest(IrrationalConstants; manual=false)
 end
 
 # copied from https://github.com/JuliaLang/julia/blob/cf5ae0369ceae078cf6a29d7aa34f48a5a53531e/test/numbers.jl
 # and adapted to irrationals in this package
 
 @testset "IrrationalConstant zero and one" begin
-  @test one(twoπ) === true
-  @test zero(twoπ) === false
-  @test one(typeof(twoπ)) === true
-  @test zero(typeof(twoπ)) === false
+    @test one(twoπ) === true
+    @test zero(twoπ) === false
+    @test one(typeof(twoπ)) === true
+    @test zero(typeof(twoπ)) === false
 end
 
 @testset "IrrationalConstants compared with IrrationalConstants" begin
-  for i in (twoπ, invπ, sqrt2, logtwo)
-    for j in (twoπ, invπ, sqrt2, logtwo)
-      @test isequal(i==j, Float64(i)==Float64(j))
-      @test isequal(i!=j, Float64(i)!=Float64(j))
-      @test isequal(i<=j, Float64(i)<=Float64(j))
-      @test isequal(i>=j, Float64(i)>=Float64(j))
-      @test isequal(i<j, Float64(i)<Float64(j))
-      @test isequal(i>j, Float64(i)>Float64(j))
+    for i in (twoπ, invπ, sqrt2, logtwo)
+        for j in (twoπ, invπ, sqrt2, logtwo)
+            @test isequal(i==j, Float64(i)==Float64(j))
+            @test isequal(i!=j, Float64(i)!=Float64(j))
+            @test isequal(i<=j, Float64(i)<=Float64(j))
+            @test isequal(i>=j, Float64(i)>=Float64(j))
+            @test isequal(i<j, Float64(i)<Float64(j))
+            @test isequal(i>j, Float64(i)>Float64(j))
+        end
```

Comment:
```suggestion
    for i in (twoπ, invπ, sqrt2, logtwo), j in (twoπ, invπ, sqrt2, logtwo)
        @test isequal(i==j, Float64(i)==Float64(j))
        @test isequal(i!=j, Float64(i)!=Float64(j))
        @test isequal(i<=j, Float64(i)<=Float64(j))
        @test isequal(i>=j, Float64(i)>=Float64(j))
        @test isequal(i<j, Float64(i)<Float64(j))
        @test isequal(i>j, Float64(i)>Float64(j))
```

Reply:
Are there any reasons to limit the test with `(twoπ, invπ, sqrt2, logtwo)`? I thought this can be replaced with `[T() for T in subtypes(IrrationalConstants.IrrationalConstant)]`.

Reply:
IMO we could also only check it for two irrationals, there's no need to test every possible combination. We don't define these functions on an instance-level but for all `IrrationalConstant`s. It's just that base uses a similar amount of irrationals to check their definitions of `<`, `<=` etc.

Reply:
I see.